client/components/app.jsx

```diff
@@ -4,40 +4,40 @@ import PartySize from './partySize.jsx';
 import Date from './date.jsx';
 import Time from './time.jsx';
 
-class App extends React.Component { 
-	constructor(props) {
-		super(props);
-		this.state = {
-			restaurantId: '',
-		}
-	} 
-	getInfo () {
-		axios.get('/restaurant/:restaurant_id')
-			.then(function(reponse) {
-				console.log(response);
-				this.setState({
-					restaurantId: response
-				})
-			}); 	
-	}
-	render () {
-		return (
-			<div id="appBox">
-				<div id="reserveBox">
-					<h3 id="title">Make a reservation</h3>
-					<div id="partySize">
-						<PartySize />
-					</div>
-				</div>
-				<div id="headers">
-					<div id="dateTitle">Date</div>
-					<div id="timeTitle">Time</div>
-					<Date />
-					<Time />
-				</div>
-			</div>
-		) 
-	}
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      restaurantId: '',
+    }
+  } 
+  getInfo () {
+    var context = this; 
```

Comment:
This works and is functional as well. If I may make a recommendation, I would suggest refactoring your function inside the '.then()" to be an arrow-function since they do not have their own this binding and you wouldn't need the extra variable. Also fits in more with your ES6 class syntax for React.